### PR TITLE
Links to references

### PR DESCRIPTION
**Pull request by gsarma**
@tpoterba @jbloom22

### CODE REVIEW

File: hail/www/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Contact the Hail team at <a href="mailto:hail@broadinstitute.org"><code>hail@bro
 
 ### Citing Hail
 
-If you use Hail for published work, please cite the software. You can get a
+If you use Hail for published work, please cite the software. We maintain a [list of publications and projects](references.md) that have used Hail and would like to know about your work. You can get a
```
Comment (jbloom22) [inline hail/www/index.md]: Suggested rewrite:
If you use Hail for published work, please cite the software and add a link to your work to our growing [list of Hail-powered science](references.md).

File: hail/www/references.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+
+# Scientific Output from the Hail Community
```
Comment (jbloom22) [inline hail/www/references.md]: I'm not sure we want to presume that everyone who uses Hail in their work considers themselves part of the Hail community. What about a title like "Hail-powered Science"?

File: hail/www/references.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+
+# Scientific Output from the Hail Community  
+
+The following is a sample of research results that have used Hail as some part of the
```
Comment (jbloom22) [inline hail/www/references.md]: Suggested rewrite: The following is an incomplete list of scientific work enabled by Hail. We welcome you to add additional examples by [editting the page directly](https://github.com/hail-is/hail/edit/master/hail/www/references.md), after which we will review the pull request to confirm the addition is valid. Please adhere to the existing formatting conventions.

File: ci/ci/constants.py [hunk 4]
```diff
@@ -22,5 +22,6 @@
     'daniel-goldstein',
     'ahiduchick',
     'GreatBrando',
-    'johnc1231'
+    'johnc1231',
```
Comment (tpoterba) [inline ci/ci/constants.py]: you'll have to rebase anyhow once Milo's change goes in (he's editing this same location) so you can either wait until that goes in and rebase, or split this out into a separate PR and we can get this one in now

Comment (gsarma) [reply ci/ci/constants.py]: Will split out.
